### PR TITLE
CI/CD and async writer

### PR DESCRIPTION
**Pull request by rsgambhir**

### CODE REVIEW

**Comment by rsgambhir** [comment]
Some of their existing tests are [failing](https://github.com/RamneekSingh24/redis-py/actions/runs/3780540830/jobs/6426714728)

[this commit](https://github.com/codenet/redis-py/commit/4200d54c748b8de7ae01ea9c0a45b149f32b5c19) in redis-py seems to suggest that there was some issue in the tests from their end, which we haven't merged up with yet.

File: tests/redis_popper/test_async_writes.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    ar.wait_all()
+
+    for i in range(100):
+        x = r.get(str(i))
+        assert int(x) == i
+
+    bench_async_writer()
+
+
+def bench_async_writer():
```
Comment (codenet) [inline tests/redis_popper/test_async_writes.py]: test_bench_async_writer

**Review by codenet — APPROVED** [review]

**Comment by codenet** [comment]
@RamneekSingh24  can you merge this? And close https://github.com/codenet/popper/issues/249?

**Comment by codenet** [comment]
@RamneekSingh24 can you merge this? And close https://github.com/codenet/popper/issues/249?

**Comment by rsgambhir** [comment]
> @RamneekSingh24 can you merge this? And close [codenet/popper#249](https://github.com/codenet/popper/issues/249)?

@codenet Sorry I missed your message dated Jan 31. 
Can you sync this forked repo with upstream? 

I am not able to do it in my repo and create a pr because it is forked from yours.

**Comment by codenet** [comment]
Hmm. There are conflicts in trying to sync. Let us merge this PR into my repo for now and close the issue?

We can sync with upstream when we pair program next.

**Comment by rsgambhir** [comment]
> Hmm. There are conflicts in trying to sync. Let us merge this PR into my repo for now and close the issue?
> 
> We can sync with upstream when we pair program next.

Ok. Please merge the PR. I am not getting the merge button.

**Pull request merged**